dockstore-client/src/main/java/io/github/collaboratory/wdl/WDLClient.java

```diff
@@ -98,16 +98,27 @@ public void downloadFiles() {
         zippedEntryFile = launcherFiles.getZippedEntry();
     }
 
-    @Override
-    public File provisionInputFiles() {
-        // Get list of input files
-        Map<String, String> wdlInputs = null;
+    /**
+     * Gets the WDL inputs of a descriptor.  This should probably be moved to a helper class.
+     *
+     * @param descriptorAbsolutePath The local descriptor's absolute path
+     * @return A map of the inputs
+     */
+    public static Map<String, String> getInputFiles(String descriptorAbsolutePath) {
```

Comment:
Put the test in the same package (not directory) and you can at least use package private or protected